src/main/java/com/ontotext/trree/plugin/mongodb/MongoDBPlugin.java

```diff
@@ -243,7 +243,7 @@ public StatementIterator interpret(long subject, long predicate, long object, lo
 		if (predicate == rdf_type && context == 0) {
 			if (object >= 0)
 				return null;
-			String suffix = Utils.matchPrefix(Utils.getString(entities, object), NAMESPACE_INST);
+			String suffix = Utils.matchPrefix(Utils.getString(entities, object), NAMESPACE_INST, NAMESPACE);
```

Comment:
We should not care about this, since context is 0, we never get to here with the specific graph